src/Chain/InputProcessor/SystemPromptInputProcessor.php

```diff
@@ -14,8 +16,13 @@
 {
     public function __construct(
         private string $systemPrompt,
+        private ?ToolBoxInterface $toolBox = null,
```

Comment:
@chr-hertel is it to implicit to add the definitions, once a toolbox is available?

Reply:
why else would you inject a toolbox?